UPGRADING.INTERNALS

```diff
@@ -62,6 +62,10 @@ PHP 8.2 INTERNALS UPGRADE NOTES
 2. Build system changes
 ========================
 
+* Identifier names for namespaced functions generated from stub files through
+  gen_stub.php have been changed. This will require changes in extensions using
+  namespaced funtions.
```

Comment:
I think it would be useful to mention that the function entry name will include the namespace separated by `_`

Reply:
@alcaeus can you please mention this to the note?